code/src/pages/products/ProductsPage.scss

```diff
@@ -39,4 +39,9 @@
     margin-top: spacing(5);
     width: 100%;
   }
+
+  &__pagination {
+    padding-top: 20px;

```

Comment:
We should use spacing scss function for padding

Reply:
Thanks, I used `spacing` function there